ChangeLog.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to
 * Custom NeighborLists can be created from a set of points using `from_points`. Distances will be calculated automatically.
 * The Box class has methods `compute_distances` and `compute_all_distances` to calculate distances between arrays of points and query points.
 * Hexatic can now compute 2D Minkowski Structure Metrics, using `weighted=True` along with a Voronoi NeighborList.
+* `freud.diffraction.DiffractionPattern` class (unstable) can be used to compute 2D diffraction patterns.
```

Comment:
Why are we referring to this as unstable?

Reply:
The API is not guaranteed to remain the same for all the freud 2.x line. In a later 2.y release, we can decide it is stable, but I am not confident enough to promise API stability at this time.